### ISSUE TITLE
Homepage nav to auth and auth to home nav button should be a link

### ISSUE THREAD

**Issue by 413ksz** [413ksz/BlueFox#26]
**Is your feature request related to a problem? Please describe.**
Homepage nav to auth and auth to home nav button should be a link for better accessibility
